.github/ISSUE_TEMPLATE/retro.md

```diff
@@ -2,7 +2,7 @@
 name: Retro
 about: Ensure all steps for Retro are completed & tracked
 title: ''
-labels: Pri1, Retro
+labels: Retro
```

Comment:
Comparing PR's, this tag wasn't removed in the helium repo - not sure if it was intentional